INSTALL.md

```diff
@@ -69,40 +69,36 @@ brew info elasticsearch
 ```
 
 ### MYSQL Database
-Ensure your database is running.
+Ensure your database is running with the following command:
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/commit/5d06e4dbdbc421dc9f5c6b64773357caedabcff9